### ISSUE TITLE
Any good alternatives for Laravel v7.0? 

### ISSUE THREAD

**Issue by RishabSwift** [leantony/laravel-grid#123]
Just working on upgrading a huge project, this package had been such a life-saver. 

But it seems like this package is no longer updated for Laravel v7.0.

Have you guys found a good alternative?

Some alternatives I found:
- [Laravel Views](https://github.com/Gustavinho/laravel-views)
- [ Laravel Livewire Tables](https://github.com/rappasoft/laravel-livewire-tables)

However, the ones I posted do not have many common features (like sorting). Please share any good alternatives for this package if you know any. 

Thanks!

**Comment by manikshrma** [comment]
Did you found any good solution?

**Comment by sunilit42** [comment]
@RishabSwift 
have you find any thing? or which one used?

**Comment by RishabSwift** [comment]
Yes, I ended up using [Livewire Powergrid](https://github.com/Power-Components/livewire-powergrid)
